src/utils.ts

```diff
@@ -0,0 +1,58 @@
+import { VALIDATION_ERROR_NAME } from './constants';
+import { Indexable } from './types';
+
+export const toInt = (str: string, defaultNum = 0) => {
+  const val = parseInt(str, 10);
+  return !Number.isNaN(val) ? val : defaultNum;
```

Comment:
개인적인 취향이지만 삼항연산자를 쓴다면 아래와 같은 방법으로 작성하는게 가독성이 좀 더 좋아보이는 것 같습니다.
```suggestion
  return Number.isNaN(val) ? defaultNum :  val ;
```

Reply:
네, 앞에 !가 붙는것보다 더 읽기가 편하네요.
수정하겠습니다 :D